sway-core/src/asm_lang/virtual_ops.rs

```diff
@@ -473,6 +477,19 @@ impl VirtualOp {
                         .collect()
                 }
             }
+            JNZI(_, i) => {
+                // Two possible successors: the next instruction as well as the instruction
+                // indicated in the jump offset. Use `offset_to_ix` to figure out the index in
+                // `ops` that corresponds to the offset specified.
```

Comment:
Is this an exact copy of `JNEI`?  If so they might as well share the same code with `JNEI(_, _, i) | JNZI(_, i) => { ... }`

Reply:
It is the same code but `JNEI(_, _, i) | JNZI(_, i) => { ... }` apparently doesn't work because the `i`s have different types (`VirtualImmediate12` v.s. `VirtualImmediate18`). Is there a way to make it work? 

I suppose I could always pull the common code out into a separate generic function.

Reply:
Oh right, of course.